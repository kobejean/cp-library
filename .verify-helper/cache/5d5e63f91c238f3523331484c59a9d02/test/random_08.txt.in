98
31671403 20418380 768509032
276714329 263121444 968795649
604291913 589974169 428621394
662307873 485922978 547767737
649951793 470330638 926476109
867289534 666667328 440891151
132267229 77475514 449897709
346830325 46195409 458837633
868414489 768041062 910162323
551349093 413794464 382527583
225940529 207083821 724893019
631609067 139292566 439772854
41481757 32049193 895367313
306811176 136496229 819883489
138812731 128005730 45183028
966137285 952042356 287647562
859169237 666538001 558765848
97208965 14195612 41677228
337748891 265571152 232027565
656506876 493074551 296810143
911552641 113488907 170840258
524563205 371863758 507743886
312308141 18144525 613638478
826768195 226754432 446625763
944324383 680825514 903427448
215898260 69548355 508333283
899160571 263113486 93701574
668768117 246577859 931004763
76643173 22504266 504696739
755977641 643945618 438102611
895170359 421292043 47345676
931306100 890500337 2400339
107644171 42672727 916098420
739036665 254396578 989090681
561623213 406094844 408689086
777889956 87038158 362373755
344410603 29970455 290437097
817502230 86146860 353293779
484207777 237790727 222347605
172158388 86137486 75408421
727630396 490575157 898411842
329580069 89985753 338336598
762905677 419337513 336068611
248984319 44914737 484285473
698349673 642901805 881733237
708267792 285807216 301065897
332399155 268422 955420450
211616387 800538 423232774
990969527 566589254 990969527
55679731 12928341 167039193
942678515 638399641 942678515
195980597 194730882 587941791
346643539 294932893 346643539
453989609 369301627 907979218
453755641 211588466 453755641
877074281 731067144 877074281
974761254 361256364 974761254
588416009 17257581 588416009
455303919 134766715 455303919
731155861 540010746 731155861
682015962 670582352 682015962
935750059 627537186 935750059
564171196 313121888 564171196
553894141 173356099 553894141
132283118 103736914 925981826
813648827 257572320 813648827
895750538 374354123 895750538
1796437 551941 826361020
495119236 306135502 990238472
111815707 54310530 223631414
242819416 78405283 485638832
998687449 255044205 998687449
529727200 186688070 529727200
45761083 2004177 549132996
747223634 655077008 747223634
431204143 190372412 431204143
262657820 245128050 525315640
963287623 533423311 963287623
491382426 2306522 982764852
75995471 5308617 75995471
545098878 265721513 545098878
238588243 236035063 715764729
403427957 162919657 806855914
720627241 87249315 720627241
855926794 371061486 855926794
204060049 88538801 612180147
18742347 4737648 187423470
654975109 600661706 654975109
43831221 3456956 438312210
56131609 45909888 112263218
107105605 24714243 428422420
999999938 10204081 51020405
999999982 894736826 631578936
1000000000 212500000 512500000
999999982 827586192 379310338
999999938 397959159 724489751
1000000000 760000000 680000000
999999924 740259684 896103828
