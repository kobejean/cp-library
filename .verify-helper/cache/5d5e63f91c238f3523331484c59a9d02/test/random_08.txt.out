25110665
46
604291851
516923469
3
867289442
16884834
77
868414429
340406106
58
631608992
25379729
99
138812636
757646857
68
97208946
287864730
47
911552635
229221842
87
826768126
316264146
95
899160549
636326355
73
755977624
306025959
17
107644133
664015897
10
777889930
316451416
90
484207691
19255150
-1
65
1633573
21161359
46
236089216
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
39
16
23
11
45
18
36
